tests/test_mutation.py

```diff
@@ -40,6 +40,32 @@ def mutated_module(source: str) -> str:
         # ('break', 'continue'),  # probably a bad idea. Can introduce infinite loops.
         ('break', 'return'),
         ('continue', 'break'),
+        ('a.lower()', 'a.upper()'),
+        ('a.upper()', 'a.lower()'),
```

Comment:
Could you add a test for `('a.b.lower()', 'a.b.upper()'),` here? Your code already correctly handles this case, but would be nice to cover in the tests.

```suggestion
        ('a.upper()', 'a.lower()'),
        ('a.b.lower()', 'a.b.upper()'),
```

Reply:
Thanks for the suggestion! It's a good idea to have explicit test coverage for nested attribute access.

Reply:
@Otto-AA 
Tests were added)
Do you want me to fix something else?